learners-guild/update.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
```

Comment:
Also using absolute paths is safer. Like this https://github.com/LearnersGuild/echo-chat/pull/13/files